Parsers/ASimNetworkSession/ARM/ASimNetworkSessionMicrosoft365Defender/README.md

```diff
@@ -0,0 +1,15 @@
+# M365 Defender ASIM NetworkSessions Normalization Parser
+
+This template deploys the ASIM NetworkSessions schema parser for M365 Defender. The parser is a part of the Advanced SIEM Information Model.
```

Comment:
Checked and the official name is "Microsoft Defender for Endpoint" (no 365, Endpoint in singular)

Reply:
I fixed the text for now, but changing the function name is more complicated